api_fhir_r4/converters/coverageEligibilityRequestConverter.py

```diff
@@ -17,10 +16,10 @@ def to_fhir_obj(cls, coverage_eligibility_response):
 
     @classmethod
     def to_imis_obj(cls, fhir_coverage_eligibility_request, audit_user_id):
-        uuid = cls.build_imis_uuid(fhir_coverage_eligibility_request)
+        chf_id = cls.build_imis_uuid(fhir_coverage_eligibility_request)
         service_code = cls.build_imis_service_code(fhir_coverage_eligibility_request)
```

Comment:
@dragos-dobre For me it make sense to use CHF_ID iso uuid but will it work with AI ?

Reply:
@delcroip We don't use the coverageEligibilityRequest in AI. I agree with you, when doing an enquiring it is better to use CHFID.